types/tabulator-tables/index.d.ts

```diff
@@ -1,8 +1,8 @@
-// Type definitions for tabulator-tables 4.5
+// Type definitions for tabulator-tables 4.6
 // Project: http://tabulator.info
 // Definitions by: Josh Harris <https://github.com/jojoshua>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 3.7
```

Comment:
It seems unfortunate to restrict the consumers of this package to only those running 3.7 or higher just because of the optional chaining usage on line 697 of the tests. Could you change that line and make this versioning more permissive? Thanks!

Reply:
I agree. I was thinking about this after I changed it. It doesn't really make sense that the minimum TS version check goes against the tests rather than the actual .d.ts file where it will be used. I've updated it.

Reply:
Another minor change but should be good to go now.